packages/canon/src/components/Heading/Heading.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { forwardRef } from 'react';
+import { HeadingProps } from './types';
+import { useTheme } from '../../theme/context';
+import { getResponsiveValue } from '../../utils/getResponsiveValue';
+
+export const Heading = forwardRef<HTMLParagraphElement, HeadingProps>(
+  (props, ref) => {
+    const { children, variant = 'title1', ...restProps } = props;
+    const { breakpoint } = useTheme();
+    const responsiveVariant = getResponsiveValue(variant, breakpoint);
+
+    console.log(breakpoint);
+    console.log(responsiveVariant);
+
+    return (
+      <p
```

Comment:
I'd kind of expected matching h* elements here. Correct levels here is gonna be important for accessibility right?

Reply:
This is now fixed. By default it will follow the variant to assign the right tag but if you want something specific for accessibility reasons you can use the `as` prop.